.github/workflows/ci.yml

```
@@ -2,52 +2,61 @@
 name: nf-core CI
 on:
   push:
-    branches:
-      - dev
+    branches: ["dev"]
   pull_request:
-    branches:
-      - dev
-      - master
+    branches: ["dev", "master"]
   release:
     types: [published]
 
 env:
   NXF_ANSI_LOG: false
 
+# cancel-in-progress: true ensures that if a new push is made to the same branch while a previous run is still in progress, the previous run is cancelled
 concurrency:
   group: "${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}"
   cancel-in-progress: true
 
 jobs:
   test-dev:
     # Only run if the pull request is against the dev branch or if the push is to the dev branch
-    if: github.base_ref == 'dev' || github.event_name == 'push'
+    if: ${{ (github.base_ref == 'dev' || (github.event_name == 'push')) && !contains(github.event.head_commit.message, '[ci skip]') }}
     name: Run pipeline with test data on dev
     runs-on: ubuntu-latest
+    # Add PR write permissions to allow commenting on the PR
+    permissions:
+      pull-requests: write
     # Set up the matrix of different pipeline versions and profiles to test
     strategy:
+      # Do not fail fast if one of the tests fails
+      fail-fast: false
       matrix:
         NXF_VER: ["latest-everything"]
+        NFTEST_VER: ["0.7.3"]
```

```suggestion
        NFTEST_VER: "0.7.3"
```

Or this could be an env variable. I went down the same path and then realized we'd never run the tests on multiple nf-test versions, one is enough! 😅 

But I like it being a variable for a cleaner template update later.